autorest/codegen/templates/templates_package/README.md.jinja2

```diff
@@ -0,0 +1,119 @@
+{% if package_mode == "mgmtplane" -%}
+# Microsoft Azure SDK for Python
+
+This is the Microsoft {{package_pprint_name}} Client Library.
+This package has been tested with Python 3.6+.
+For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+# Usage
+
+To learn how to use this package, see the [quickstart guide](https://aka.ms/azsdk/python/mgmt)
+
+For docs and references, see [Python SDK References](https://docs.microsoft.com/python/api/overview/azure)
+Code samples for this package can be found at [{{package_pprint_name}}](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com.
+Additional code samples for different Azure services are available at [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
+
+# Provide Feedback
+
+If you encounter any bugs or have suggestions, please file an issue in the
+[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
+section of the project. 
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2F{{package_name}}%2FREADME.png)
+{% else %}
+# {{ package_pprint_name }} client library for Python
+<!-- write necessary description of service -->
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please 
+refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install {{ package_name }}
```

Comment:
nit: you only need to do `pip install {{ package_name }}`

also does mgmt plane need this information too, or is that handled differently in mgmt plane?

Reply:
mgmt plane has this [template](https://github.com/Azure/azure-sdk-for-python/blob/main/tools/azure-sdk-tools/packaging_tools/templates/README.md). Even if we add it in autorest.python template, it will be rerendered once the pr is merged into main branch. So I think we just need to keep same between autorest.python and SDK repo about mgmt template.

Reply:
Sorry I'm confused, is mgmt plane still planning on running the packaging_tools command line with this flag too? I thought the poitn of `--package-mode` was so we didn't need to run packaging tools scripts after generation.

Since `python -m pip install` is not really different from `pip install` it's not a hill i'm willing to die on, but the data plane sdks have standardized using just pip install